mapquest_geocoder.go

```diff
@@ -89,7 +98,11 @@ func (g *MapQuestGeocoder) extractLatLngFromResponse(data []byte) (Point, error)
 // Returns the first most available address that corresponds to the passed in point.
 // It may also return an error if one occurs during execution.
 func (g *MapQuestGeocoder) ReverseGeocode(p *Point) (string, error) {
-	data, err := g.Request(fmt.Sprintf("reverse.php?lat=%f&lon=%f&format=json", p.lat, p.lng))
+	key := ""
+	if (GoogleAPIKey != "") {
+		key = "&key="+GoogleAPIKey+"&"
+	}
+	data, err := g.Request(fmt.Sprintf("reverse.php?%slat=%f&lon=%f&format=json", key, p.lat, p.lng))
```

Comment:
Yep this should yield something like 
`reverse.php?key=hdufhhsfsuhfhf&lat=9999.0&lon=99999.0&format=json`

or if key is not set

`reverse.php?lat=9999.0&lon=99999.0&format=json`

should the format if there is a blank key be? 
`reverse.php?key=&lat=9999.0&lon=99999.0&format=json`